### DISCUSSION TITLE
Images not working in MDX files

### DISCUSSION THREAD

**Discussion by argdesign** [storybookjs/storybook#21624] [Help] [answered]
Hi! I'm trying to use images in my mdx files but for some reason it's not working. I follow the documentation https://storybook.js.org/docs/angular/configure/images-and-assets

**main.s**

```
module.exports = {
  "stories": [
    "../src/**/*.stories.mdx",
    "../src/**/*.stories.@(js|jsx|ts|tsx)"
  ],
 ...
  staticDirs: ['../src/stories/assets'],
...
}
```

**mdx file**
`<img src="/assets/colors.svg" alt="alt"/>`

**my files structure**
```
- .storybook
- Src 
  —stories
  ---—assets
```

Thanks in advance

**Comment by bzh2610** [comment, marked as answer]
Hi, could you try to specify explicitly the from and to parameters ? 

```
  staticDirs: [{ from: "../src/stories/assets", to: "/assets" }],
```

Also, which framework are you using ?
On Angular you need to specify the assets directory in your angular.json file so that this works properly: https://angular.io/guide/workspace-config#asset-config

I'm no sure about others :)

**Comment by argdesign** [comment]
I'm using Angular, it worked! thanks !!
